src/main/java/com/google/codeu/data/Item.java

```diff
@@ -0,0 +1,36 @@
+package com.google.codeu.data;
+
+public class Item {
+
+  private String title;
+  private Double price;
+  private String email;
+  private String description;
+
+  /* Item class. Stores information about a specific item posted by a user */
```

Comment:
What stickler is complaining about is you need a /** instead of a /*, the extra * denotes a javadoc comment